### ISSUE TITLE
Make workflow/group helptext reachable via keyboard

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#6554]
**Current behavior**
Helptext from workflow and group permissions are not reachable via  keyboard

**Wanted behavior**
Helptexts should be reachable via keyboard

**Additional context**

<img width="346" height="167" alt="Image" src="https://github.com/user-attachments/assets/be71bb71-673e-4745-bbd9-3af94664e6d6" />

**Comment by GhostA11yDev** [comment]
Hey! 👋 I noticed this project is actively working on accessibility (ADA compliance).

If you want to save time, you can use our free GitHub App [GhostA11y Auto-Fix](https://github.com/marketplace/ghosta11y-auto-fix). It automatically scans Pull Requests and injects missing `aria-labels` and `alt` tags using Context-Aware AI.

Hope this saves your developers some manual QA work! 👻

**Comment by bspekker** [comment]
Added helptexts in group permissions to issue description (could also be an additional PR/issue)
